### PR TITLE
Renamed "overview" to "reference card"

### LINKED ISSUE #550: Consider renaming overview PDFs?
When reading through [next/README.md](https://github.com/CesiumGS/3d-tiles/tree/main/next), which arguably looks a lot like an overview of 3D Tiles Next extensions, it's a bit confusing to find this link at the bottom of the page:

> The [3D Tiles Next Overview](https://github.com/CesiumGS/3d-tiles/blob/main/next/3d-tiles-next-overview.pdf) is an overview of the new extensions that are introduced with 3D Tiles Next.

In Khronos terminology these are "Reference Guides" or "Reference Cards", I wonder if we should adopt similar naming here? See: https://www.khronos.org/developers/reference-cards/. I was trying to come up with a better way to differentiate the overview wording, like "an overview with more detail than this overview but less detail than the specifications", but didn't get very far with that. :)

### PR DESCRIPTION
**Pull request by javagl**
Fixes #550 

The "overview" of 3D Tiles **and** 3D Tiles Next are now renamed to "reference card".

# Note: Merging this may cause some links to break!

Specifically, I think that _at least_ the blog posts should be updated:

- The blog post for 3D Tiles Next: https://cesium.com/blog/2021/11/10/introducing-3d-tiles-next 
- The blog post announcing the 3D Tiles Overview: https://cesium.com/blog/2020/03/12/3d-tiles-overview/

@ptrgags What's the best way for syncing this PR and the update of these blog posts?

### CODE REVIEW

File: reference-card/README.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ This directory contains the source files for generating the 3D Tiles reference c
...
+The `/resources` subdirectory contains the images (screenshots and SVG versions of the logos) that are used in the reference card. They are not necessary for creating the PDF files, but are added here for reference.
 
-The file `/resources/3d-tiles-overview-tables.odt` is an [OpenDocument Text](https://en.wikipedia.org/wiki/OpenDocument) that can be edited with any standard office program. It contains most of the tables that are used in the overview. In order to change one of the tables, it is edited in the ODT file, exported as PDF, and imported into Inkscape. (Considering the limited support for tables in Inkscape, this still seemed to be the most simple approach here.)
+The file `/resources/3d-tiles-reference-card-tables.odt` is an [OpenDocument Text](https://en.wikipedia.org/wiki/OpenDocument) that can be edited with any standard office program. It contains most of the tables that are used in the reference -card. In order to change one of the tables, it is edited in the ODT file, exported as PDF, and imported into Inkscape. (Considering the limited support for tables in Inkscape, this still seemed to be the most simple approach here.)
```
Comment (ptrgags) [inline reference-card/README.md:27]:
```suggestion
The file `/resources/3d-tiles-reference-card-tables.odt` is an [OpenDocument Text](https://en.wikipedia.org/wiki/OpenDocument) that can be edited with any standard office program. It contains most of the tables that are used in the reference card. In order to change one of the tables, it is edited in the ODT file, exported as PDF, and imported into Inkscape. (Considering the limited support for tables in Inkscape, this still seemed to be the most simple approach here.)
```

**Review by ptrgags — COMMENTED** [review]
@javagl looks good, noticed one small typo though.

**Comment by ptrgags** [comment]
@javagl looks good, I'm merging. I'll talk to @AndieTursi today about updating the blog so it points to the updated link

**Comment by AndieTursi** [comment]
@ptrgags text and link changes were made to the following pages:

- https://cesium.com/why-cesium/3d-tiles/ 
- https://cesium.com/blog/2020/03/12/3d-tiles-overview/ 
- https://cesium.com/blog/2021/11/10/introducing-3d-tiles-next 

It may take up to an hour for these changes to be live on Cesium.com.

**Pull request merged**